touchforms/backend/touchcare.py

```diff
@@ -187,8 +187,9 @@ def _get_fixture(self, user_id, fixture_id):
                 raise TouchFormsNotFound('This form requires that the user be assigned to a location '
                                          'but one could not be found.')
             else:
-                raise TouchFormsNotFound('Unable to fetch fixture %s. '
-                                         'Ensure the logged in user has access to this fixture.' % fixture_name)
+                raise TouchFormsNotFound('Unable to fetch lookup table %s. '
+                                         'Ensure the logged in user has access '
+                                         'to this lookup table.' % fixture_name)
```

Comment:
can we search for `item-list` the same way we special case the other ones? other fixtures (products, programs, ucrs) won't actually be lookup tables